tools/GenerateCompilationDatabase.ps1

```diff
@@ -0,0 +1,166 @@
+Param(
+    [string]$Ms2ccVersion = "1.3.0",
+    [switch]$Update
+
+)
+
+$binlogFileBase = Join-Path (Split-Path $PSScriptRoot -parent) "BuildOutput\Binlogs"
+$binlogFiles = Get-ChildItem -Path $binlogFileBase -Filter *.binlog -Recurse | Select-Object -ExpandProperty FullName
+
+Write-Host "Binlog files found:"
+$binlogFiles | ForEach-Object { Write-Host $_ }
+
+$VCToolsInstallDir = . "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -Latest -prerelease -requires Microsoft.Component.MSBuild -property InstallationPath
+write-host "VCToolsInstallDir: $VCToolsInstallDir"
```

Comment:
Is this meaningful to users of the script?

Such things can be useful for debugging scenarios but not so much for common usage.

SUGGESTION: `Write-Verbose` instead of `Write-Host`. Then folks can see this added info when they want if they specify the `-Verbose` command line option